xml_inscriptions/bib_only/MO.Col.UM.MAA.L.59.2.xml

```diff
@@ -62,7 +62,7 @@
                <bibl>
                   <ref type="external"
                      target="http://maacollections.missouri.edu/ArgusNET/Portal.aspx?lang=en-US&amp;g_AAEM=Arretine+Sigillata+Plate+59.2"
-                     >Museum of Art and Archeology (Web Entry)</ref>
+                     >Museum of Art and Archeology</ref>
```

Comment:
Any reason we're getting rid of this second piece here, rather than changing to something like (Online catalog entry)? At any rate we should probably put the entire museum name here to make it clear to what the link will actually lead.

Reply:
originally the reference was showing up as "Museum of Art and Archeology (Web Entry) (external reference)"; the last part seemed to be added automatically, so I deleted "(Web Entry) because it felt redundant. However it seems like it's not showing up that way anymore (maybe something to do with the recent bibliography changes) so I'll go back and re-add it.

Reply:
Yeah, I think that's what's happened here; we can do it either way, honestly, we'll just want to make a decision and stick with it one way or the other. Might be a bigger conversation more generally so leaving it however you prefer for the time being is fine, let's just make note of which it is if/when we need to make any global changes.